g++ warnings in test/unit/TestInstrumentation.cpp

Using g++ on Linux I get the following warnings:

```
/home/etiotto/Source/onnx/onnx-mlir/test/unit/TestInstrumentation.cpp:18:20: warning: array subscript ‘const int64_t {aka const long int}[0]’ is partly outside array bounds of ‘const char [5]’ [-Warray-bounds]
   18 |   OMInstrumentPoint(*(const int64_t*)op3, 4);
      |   ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
/home/etiotto/Source/onnx/onnx-mlir/test/unit/TestInstrumentation.cpp:12:14: note: while referencing ‘op3’
   12 |   const char op3[]     = "TOp3";
      |              ^~~
/home/etiotto/Source/onnx/onnx-mlir/test/unit/TestInstrumentation.cpp:19:20: warning: array subscript ‘const int64_t {aka const long int}[0]’ is partly outside array bounds of ‘const char [5]’ [-Warray-bounds]
   19 |   OMInstrumentPoint(*(const int64_t*)op4, 9);
      |   ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
/home/etiotto/Source/onnx/onnx-mlir/test/unit/TestInstrumentation.cpp:13:14: note: while referencing ‘op4’
   13 |   const char op4[]     = "TOp4";
      |              ^~~
/home/etiotto/Source/onnx/onnx-mlir/test/unit/TestInstrumentation.cpp:21:20: warning: array subscript ‘const int64_t {aka const long int}[0]’ is partly outside array bounds of ‘const char [7]’ [-Warray-bounds]
   21 |   OMInstrumentPoint(*(const int64_t*)opfinal, 12);
      |   ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/etiotto/Source/onnx/onnx-mlir/test/unit/TestInstrumentation.cpp:14:14: note: while referencing ‘opfinal’
   14 |   const char opfinal[] = "TOpFin";
```

--- a/src/Dialect/ONNX/MLIRDialectBuilder.cpp
+++ b/src/Dialect/ONNX/MLIRDialectBuilder.cpp
@@ -270,8 +270,8 @@ Value MathBuilder::positiveInf(Type type) const {
                              : std::numeric_limits<uint32_t>::max();
           break;
         case 64:
-          value = (isSigned) ? std::numeric_limits<int64_t>::max()
-                             : std::numeric_limits<uint64_t>::max();
+          value = static_cast<double>((isSigned) ? std::numeric_limits<int64_t>::max()
+                                                 : std::numeric_limits<uint64_t>::max());
           break;
         default:
           llvm_unreachable("unsupported element type");

--- a/src/Dialect/ONNX/MLIRDialectBuilder.cpp
+++ b/src/Dialect/ONNX/MLIRDialectBuilder.cpp
@@ -270,8 +270,9 @@ Value MathBuilder::positiveInf(Type type) const {
                              : std::numeric_limits<uint32_t>::max();
           break;
         case 64:
-          value = static_cast<double>((isSigned) ? std::numeric_limits<int64_t>::max()
-                                                 : std::numeric_limits<uint64_t>::max());
+          value = static_cast<double>(
+              (isSigned) ? std::numeric_limits<int64_t>::max()
+                         : std::numeric_limits<uint64_t>::max());
           break;
         default:
           llvm_unreachable("unsupported element type");

--- a/test/perf/PerfConv.cpp
+++ b/test/perf/PerfConv.cpp
@@ -53,4 +53,4 @@ BENCHMARK(BM_Conv2D_C16_K3)
     ->ArgsProduct({{1, 16, 64}, {16, 64, 256}})
     ->Unit(benchmark::kMillisecond);
 
-PERF_MAIN();
+PERF_MAIN()

--- a/test/perf/PerfGemm.cpp
+++ b/test/perf/PerfGemm.cpp
@@ -86,4 +86,4 @@ BENCHMARK(BM_GemmSquare)
     ->Unit(benchmark::kMillisecond)
     ->Complexity();
 
-PERF_MAIN();
+PERF_MAIN()

--- a/test/unit/TestInstrumentation.cpp
+++ b/test/unit/TestInstrumentation.cpp
@@ -7,11 +7,11 @@
 #include <thread>
 
 int main(int argc, char *argv[]) {
-  const char opstart[] = "TOpStar";
-  const char op2[]     = "TOp2";
-  const char op3[]     = "TOp3";
-  const char op4[]     = "TOp4";
-  const char opfinal[] = "TOpFin";
+  const char opstart[8] = "TOpStar";
+  const char op2[8]     = "TOp2";
+  const char op3[8]     = "TOp3";
+  const char op4[8]     = "TOp4";
+  const char opfinal[8] = "TOpFin";
   OMInstrumentInit();
   OMInstrumentPoint(*(const int64_t*)opstart, 13);
   OMInstrumentPoint(*(const int64_t*)op2, 1);

